### ISSUE TITLE
Array to string conversion warning for "custom" link attributes

### ISSUE THREAD

**Issue by uuf6429** [storyblok/storyblok-php-richtext-renderer#14]
[storyblokurl]: https://www.storyblok.com?utm_source=github.com&utm_medium=issue-template

Links with custom attributes cause an error in the renderer.

---

## Expected Behavior

Link HTML should be rendered correctly.

## Current Behavior

Renderer fails:
> ErrorException: Array to string conversion in vendor/storyblok/richtext-resolver/src/Utils
/Render.php:47

## Reproduction

1. Create a link with some custom attributes, like below:
<img width="553" alt="image" src="https://user-images.githubusercontent.com/230049/207282856-7c0ef25f-7bb9-4b18-a7e6-4dd9ed2552e1.png">
2. Have this library render the resulting json.

### Solution

Before the line 45 (`foreach ($tag['attrs'] as $key => $value) {`) add the following code:
```php
                if (isset($tag['attrs']['custom']) && is_array($tag['attrs']['custom'])) {
                    $tag['attrs'] = array_merge($tag['attrs'], $tag['attrs']['custom']);
                    unset($tag['attrs']['custom']);
                }
```

**Comment by Noaber** [comment]
Having the same issue. Is there is any timeframe when this issue is resolved ?

**Comment by roberto-butti** [comment]
Thanks to the great effort by @uuf6429 , with #15 and #18 we can close this one.
Do you agree @uuf6429 ?


@Noaber, i don't know if you can test your use case with the master branch

```
composer require storyblok/richtext-resolver dev-master
```

Or if you can't use a "not yet published version", I would ask you to copy and paste the JSON snippet of your rich text and try to describe your expectation in terms of generated HTML. In this case, we can add a new test case to the test suite.

**Comment by uuf6429** [comment]
@roberto-butti today I found out two things that we should fix with the new link code:
1. move the "custom" attributes handling to get_link instead of where it is now (since it relates specifically to links)
2. apparently we get some empty custom attributes by default that cannot be removed, so the code should filter them out instead of adding empty attributes (for example rel, title and (for some reason) onclick)

I'll open a PR later on today.

**Comment by Noaber** [comment]
@roberto-butti , the dev-master did the trick :)

**Comment by roberto-butti** [comment]
> @roberto-butti , the dev-master did the trick :)

Thank you @Noaber  for the feedback

**Comment by uuf6429** [comment]
@roberto-butti will there be an actual release with these fixes some time soon?

**Comment by roberto-butti** [comment]
> @roberto-butti will there be an actual release with these fixes some time soon?

Good question Christian @uuf6429 . In my opinion, it is fine to create a new release with the latest fixes.
Let me check with the SDK team, I'm going to ask to create a new release.

Thank you!
I will update you.

**Comment by roberto-butti** [comment]
Hi @uuf6429 we just released version 2.1.0 
https://github.com/storyblok/storyblok-php-richtext-renderer/releases/tag/2.1.0
